flyway.yaml

```diff
@@ -0,0 +1,386 @@
+package:
+  name: flyway
+  version: 11.1.1
+  epoch: 0
+  description: "Flyway is a database migration tool to evolve your database schema easily and reliably across all your instances."
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - bash
+      - openjdk-17-default-jvm
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - git
+      - maven
+      - openjdk-17
+      - wolfi-base
+  environment:
+    LANG: en_US.UTF-8
+    JAVA_HOME: /usr/lib/jvm/java-17-openjdk
+
+vars:
+  HdrHistogram: "HdrHistogram"
```

Comment:
All of these could exist in a few vars that's looped over instead of setting each